test-data/unit/check-isinstance.test

```diff
@@ -911,7 +911,8 @@ def foo() -> Union[int, str, A]: pass
 
 def bar() -> None:
     x = foo()
-    x + 1  # E: Unsupported left operand type for + (some union)
+    x + 1  # E: Unsupported left operand type for + (some union) \
+           # E: Unsupported operand types for + (likely involving Union)
```

Comment:
Yeah, this is unfortunate, but this wasn't trivial to fix. I'll create a follow-up issue to address the duplicate messages (and maybe generally improve error messages related to union types0.